client/state/themes/test/actions.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * External dependencies
+ */
+import sinon from 'sinon';
+import { expect } from 'chai';
+import { Map } from 'immutable';
+
+/**
+ * Internal dependencies
+ */
+import {
+	THEME_ACTIVATE_REQUEST,
+	THEME_ACTIVATE_REQUEST_SUCCESS,
+	THEME_ACTIVATE_REQUEST_FAILURE,
+} from 'state/action-types';
+import {
+	themeActivation,
+	themeActivated,
+	themeActivationFailed,
+	themeActivationSuccess,
+	activateTheme,
+} from '../actions';
+import useNock from 'test/helpers/use-nock';
+
+describe( 'actions', () => {
+	const spy = sinon.spy();
+
+	beforeEach( () => {
+		spy.reset();
+	} );
+
+	describe( '#themeActivation()', () => {
+		it( 'should return an action object', () => {
+			const themeId = 'twentysixteen';
+			const siteId = 2211667;
+			const expected = {
+				type: THEME_ACTIVATE_REQUEST,
+				themeId,
+				siteId,
+			};
+
+			const action = themeActivation( themeId, siteId );
+			expect( action ).to.eql( expected );
```

Comment:
Since deep.equal === to.eql I am OK with changing it.
I was the other way around: I first learned deep.equal and then was introduced to eql as a short hand.